src/components/forms/formComponents.js

```diff
@@ -247,9 +236,99 @@ export const formInputCreatable = (config) => {
       return isStringArr ? option.displayValue : option;
     },
     value: formValue,
-    ...creatableConfig,
+    ...selectConfig,
     'aria-describedby': ariaDescribedby
-  });
+  }) ;
+};
+
+export const formInputRadioGroup = (config) => {
+  /* options example:
+    [
+      {
+        name: 'open_access',
+        text: 'Open Access'
+      },
+      {
+        id: 'closed_access', // can specify id, uses name as id otherwise
+        name: 'closed_access',
+        text: 'Closed Access',
+      },
+      {
+        name: 'other',
+        text: 'Other',
+        type: 'string',
+        placeholder: 'Please specify.',
+      }
+    ]
+    if type is 'string', then a textbox is rendered
+    when checked.
+  */
+
+  const {
+    id, disabled, error,
+    formValue, options, ariaDescribedby,
+    setError
+  } = config;
+
+  return div({},
+    [
+      div(
+        {
+          className: 'radio-group',
+          id: id,
+        },
+        options.map(((option, idx) => {
+          const optionId = (!isNil(option.id) ? option.id : option.name);
+
+          return div({
+            style: {
+              key: idx,
+              margin: '1.0rem 0 0.5rem 0',
+            }
+          }, [
+            h(RadioButton, {
+              id: id+'_'+optionId,
+              name: id+'_'+optionId,
+              key: idx,
+              defaultChecked: !isNil(formValue) && formValue.selected === option.name,
+              onClick: () => onFormInputChange(config, {
+                selected: option.name,
+                value: (option.type === 'string'? '' : true),
```

Comment:
I think the boolean here is throwing me off a little - when would a radio button be a boolean value? I feel like then that means we are using the wrong form element, because it should be either a checkbox or a slider.

UX advantage of the slider/checkbox would be that the user doesnt have to move their mouse to change their selection. Accessibility for keyboard-only users would not have to tab between different items, and it's also easier for teh screen reader to display the status of the selection.

I think this is maybe something to consider! Maybe it would be worth just making the radiogroup values always a string? And then if there is a fallout boolean effect, we can pass through an onchange method that would update a separate state variable with the true/false.

Reply:
ok yeah so i understand how this is confusing. i mean the the OPTION itself is boolean, but the radio group isn't necessarily boolean. if that makes sense. so, for example, in a group of buttons, you might have:

- Option A (boolean) <- A is either selected or not selected, thus boolean
- Option B (boolean) <- B is either selected or not selected, thus boolean
- Other (string) <- Other must be specified, thus string

The string option adds a checkbox once clicked.

So, if you click A, it will report that A was set true. If you select Other it will report that Other was set to an empty string. 

Again, not sure if that's the best naming convention, but that was my logic behind it!

Reply:
One thing we could do is only have the `value` field when the option is a string. That might clarify your confusion.

So, if A selected: `{selected:'Option A'}`, if Other selected: `{selected: 'Other', value: ''}`. Thoughts?

Reply:
I like how the last way works `{selected: "option A" / "Other"}`! That's how i'd expect it to work vs the booleans, since the selected value should be updating a single field, not multiple